beancount/parser/booking_full_test.py

```diff
@@ -1457,6 +1465,55 @@ def test_augment__from_empty__incomplete_cost__with_currency(self, entries, __):
             CostSpec(MISSING, None, 'USD', datetime.date(2015, 10, 1), None, False),
             postings[0].cost)
 
+    @book_test(Booking.STRICT)
+    def test_augment__existing_inventory_merge(self, _, __):
+        """
+        2015-10-01 * #ante
+          Assets:Account          1 HOOL {1 USD}
+          Assets:Account          1 HOOL {3 USD}
+
+        2015-10-01 * #apply
+          ; This inventory is merged with all existing matching inventory into a single
+          ; aggregate lot.
+          Assets:Account          1 HOOL {5 USD, *}
+
+        2015-10-01 * #booked
+          Assets:Account          1 HOOL {5 USD, 2015-10-01}
+          Assets:Account          -1 HOOL {1 USD, 2015-10-01}
```

Comment:
I would insert some metadata, or at least a special flag, on all the inserted merging postings.

Reply:
Done